src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -14,7 +14,7 @@ public static partial class Http
 
             public static string SecureHost => GetValue("COREFX_SECUREHTTPHOST", DefaultAzureServer);
 
-            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "http2.akamai.com");
+            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "www.microsoft.com");
```

Comment:
I agree with @stephentoub : We should leave the test in place, pinning down the behavior of WinHTTP and the potential XPlat difference.